doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_custom_pcb.rst

```diff
@@ -110,41 +113,53 @@ The presence of a ``bicr`` node in the application devicetree will automatically
 Flashing the BICR binary
 ========================
 
+##TODO
+
 After the |NCS| build system generates the BICR binary, you must flash this binary manually.
 The content of BICR should be loaded to the SoC only once and should not be erased nor modified unless the PCB layout changes.
-To manually flash the generated :file:`bicr.hex` file to the SoC, use nRF Util as follows::
+To manually program the generated :file:`bicr.hex` file to the SoC, use nRF Util as follows::
+
+    nrfutil device program --options chip_erase_mode=ERASE_NONE --firmware bicr.hex --core Application --serial-number <serial_number>
+
+You only need to follow this programming process once, assuming the PCB configuration applied through the BICR is correct the first time.
+However, it is also possible to reprogram the BICR while in the LCS ``RoT``.
+This can be useful, for example, when adjusting the configuration as the PCB design gets refined or modified, requiring the process to be repeated.
+
+Validate the BICR binary
+------------------------
+
+After programming the BICR binary onto the device, you should validate whether the BICR was successfully installed and executed.
 
-    nrfutil device program --options chip_erase_mode=ERASE_NONE --firmware bicr.hex` --core Secure --serial-number <serial_number>
+Reset the device
+   The simplest way to check if the BICR is functioning is to reset the device.
+   If the device is still operational after the reset, the BICR is at least partially loaded.
```

Comment:
The device will be operational if the BICR is obviously wrong, so I do not think it is a good indication. And the problem is that a lot of BICRs are valid and correct but not supported. right now we only support 2 different power supplies option: (1) VDDH > 2.2V and inductor present and (2) VDDH shorted to VDD at 1.8V and inductor present. They should choose one of these two, make the board like this and choose one of two BICRs. 

A simple check could be done by seeing if it is one of these two.

Reply:
I removed this reference and added your power supply suggestions in the prepare your pcb section